### ISSUE TITLE
m3u generation?

### ISSUE THREAD

**Issue by AnotherLife** [pascl/pbPSCReAlpha#12]
I read this issue and it seems it would improve disc switching. Earlier I was playing a 2 disc game and had to use that "append" option to browse for and load the other disc

**Comment by AnotherLife** [comment]
Forgot to paste the link :D

https://github.com/pathartl/BleemSync/issues/296

**Comment by esmith13** [comment]
whoops. opened new ticket before I saw this. below is copied from the ticket I opened:

the biggest advantage of bleemsync 1.0+ is to be able to use the stock UI but RetroArch as the emulator. Bleemsync supports correct usage of RetroArch's 'Disk Control' feature if in the numbered game folder a '_BASENAME_.m3u' file exists. This also guarantees the same memory card is used for all discs of the same game.

**Scenario:
Here is the current contents of a game folder on 'SONY\games'**
.pcsx    (FOLDER)
SLUS-1234.bin    (DISC1)
SLUS-1234.cue    (CUE1)
SLUS-1235.bin    (DISC2)
SLUS-1235.cue    (CUE2)
SLUS-1234.png    (COVER ART)

If you add the file SLUS-1234.m3u with contents that are just 1 text line for each cue file in disc order, Bleemsync will load THAT instead of SLUS-1234.cue so that 'Disk Control' will function.

**Contents of SLUS-1234.m3u:**    
----(BEGINNING OF TEXT FILE)----
SLUS-1234.cue
SLUS-1235.cue
----(END OF TEXT FILE)----
